README.md

```diff
@@ -105,6 +109,35 @@ see the [cheatsheet](https://github.com/turion/rhine/blob/master/CHEATSHEET.md).
 * https://github.com/turion/rhine-tutorial: Presentation and tutorial app
 * https://github.com/turion/sonnendemo: Demo application
 
+
+## FAQ
+
+* Why does my blocking code, eg `arrMCl readLn`, behave [erratically](https://github.com/turion/rhine/issues/153)?
+
+Clocks must be the only thing that blocks the thread, not `ClSF`s. So for example, you can fix:
+
+```haskell
+arrMCl readLn
+```
+
+by using:
+
+```haskell
+tagS >>> arr read :: ClSF IO StdinClock () Int
+```
+
+`tagS` contains the string that the `StdinClock` grabbed from `stdin`, and only the clock has been allowed to block the thread!
+
+
+* Can a sampling schedule dynamically change, eg depend on a signal?
+
+Yes, for instance you could implement a distance-dependent [collision detector](https://github.com/turion/rhine/issues/152).
+
+
+* How to handle slow computations, IE computations that take longer than the sample rate?
+
+Several [strategies exist](https://github.com/turion/rhine/issues/151) and it depends on your use case. For `FixedStep` clocks, it won't matter since the execution of the program isn't tied to a realtime clock. For `ClSF`s running on `UTCTime` clocks, you can execute the slow code in a separate thread and coordinate merging the results back into the signal network.
```

Comment:
```suggestion
Several [strategies exist](https://github.com/turion/rhine/issues/151) and it depends on your use case.
For `FixedStep` clocks, it won't matter since the execution of the program isn't tied to a realtime clock.
For `ClSF`s running on `UTCTime` clocks, you can execute the slow code in a separate thread and coordinate merging the results back into the signal network.
```

Reply:
(I prefer short, semantic lines. It makes diffs easier to read and merge.)